flake.nix

```diff
@@ -11,7 +11,7 @@
 
       lib = import ./lib;
 
-      systems = lib.systems.supported.hydra;
+      systems = lib.platforms.all;
```

Comment:
I'm 90% sure that this should not be changed. `systems` is like the `system` field of derivations (= the `system`/`localSystem` argument to nixpkgs) meaning the platform the derivation is *built on*. Consequently `lib.systems.supported.hydra` is Tier 3 and above, meaning all platforms for which a native `stdenv` is available.

For a jobset `system` can't be `sparc64-linux` atm anyways, as a native `stdenv` is not available, but rather `pkgsCross` needs to be used or `crossSystem` passed (which is not possible with flakes however as far as I know).

Reply:
This change doesn't seem necessary given that `sparc64-linux` is only built using `pkgsCross` as @sternenseemann said.

The main thing you want to check is to see if `nix flakes check` keeps working after the change. It might also cause more/slower evaluation to have all the systems available.